### PR TITLE
Fix all errors in the Makefile

### PR DESCRIPTION
**Pull request by oleorhagen**
The Makefile on the master branch had several errors present, so that:

* make check
* make coverage
* make test

all failed.

This PR fixes all these issues

### CODE REVIEW

**Comment by oleorhagen** [comment]
Already greenlit by Kristian in https://github.com/mendersoftware/mender-cli/pull/58, so merging

File: Makefile [hunk 1]
```diff
@@ -85,7 +85,7 @@ coverage:
 		rm -f coverage-tmp.txt;  \
 		$(GO) test -coverprofile=coverage-tmp.txt $$p ; \
 		if [ -f coverage-tmp.txt ]; then \
-			cat coverage-tmp.txt |grep -v 'mode:' >> coverage.txt; \
+			cat coverage-tmp.txt |  grep -v 'mode:' >> coverage.txt || /bin/true; \
```
Comment (lluiscampos) [inline Makefile:88]: Is it two spaces better than 0? :P

Comment (oleorhagen) [reply Makefile:88]: haha, shite :see_no_evil:

Comment (oleorhagen) [reply Makefile:88]: https://github.com/mendersoftware/mender-cli/pull/60

**Pull request merged**